src/api/admin.ts

```diff
@@ -27,10 +27,13 @@ import { VenueTemplate } from "types/VenueTemplate";
 import { WithId, WithoutId, WithWorldId } from "utils/id";
 import { generateAttendeeInsideUrl } from "utils/url";
 
+import { SpaceType } from "components/organisms/TimingEventModal/TimingEventModal";
```

Comment:
Not sure why this is defined in one component if it is needed for the props of two different ones. Maybe there is a place in `./src/types/` for it